packages/fiori/src/NotificationListGroupItem.ts

```diff
@@ -0,0 +1,246 @@
+import Popover from "@ui5/webcomponents/dist/Popover.js";
+import NotificationListItemBase from "./NotificationListItemBase.js";
+
+// Icons
+import "@ui5/webcomponents-icons/dist/navigation-right-arrow.js";
+import "@ui5/webcomponents-icons/dist/overflow.js";
+import "@ui5/webcomponents-icons/dist/decline.js";
+
+// Texts
+import {
+	NOTIFICATION_LIST_GROUP_ITEM_TXT,
+	NOTIFICATION_LIST_GROUP_ITEM_COUNTER_TXT,
+	NOTIFICATION_LIST_ITEM_READ,
+	NOTIFICATION_LIST_ITEM_UNREAD,
+	NOTIFICATION_LIST_ITEM_HIGH_PRIORITY_TXT,
+	NOTIFICATION_LIST_ITEM_MEDIUM_PRIORITY_TXT,
+	NOTIFICATION_LIST_ITEM_LOW_PRIORITY_TXT,
+	NOTIFICATION_LIST_ITEM_OVERLOW_BTN_TITLE,
+	NOTIFICATION_LIST_GROUP_ITEM_CLOSE_BTN_TITLE,
+	NOTIFICATION_LIST_GROUP_ITEM_TOGGLE_BTN_COLLAPSE_TITLE,
+	NOTIFICATION_LIST_GROUP_ITEM_TOGGLE_BTN_EXPAND_TITLE,
+} from "./generated/i18n/i18n-defaults.js";
+
+// Templates
+import NotificationListGroupItemTemplate from "./generated/templates/NotificationListGroupItemTemplate.lit.js";
+
+// Styles
+import NotificationListGroupItemCss from "./generated/themes/NotificationListGroupItem.css.js";
+
+/**
+ * @class
+ *
+ * <h3 class="comment-api-title">Overview</h3>
+ * The <code>ui5-li-notification-group</code> is a special type of list item,
+ * that unlike others can group items within self, usually <code>ui5-li-notification</code> items.
+ * <br>
+ *
+ * The component consists of:
+ * <ul>
+ * <li><code>Toggle</code> button to expand and collapse the group</li>
+ * <li><code>Priority</code> icon to display the priority of the group</li>
+ * <li><code>TitleText</code> to entitle the group</li>
+ * <li>Custom actions - with the use of <code>ui5-notification-action</code></li>
+ * <li>Items of the group</li>
+ * </ul>
+ *
+ * <h3>Usage</h3>
+ * The component can be used in a standard <code>ui5-list</code>.
+ *
+ * <h3>CSS Shadow Parts</h3>
+ *
+ * <ui5-link target="_blank" href="https://developer.mozilla.org/en-US/docs/Web/CSS/::part">CSS Shadow Parts</ui5-link> allow developers to style elements inside the Shadow DOM.
+ * <br>
+ * The <code>ui5-li-notification-group</code> exposes the following CSS Shadow Parts:
+ * <ul>
+ * <li>title-text - Used to style the titleText of the notification list group item</li>
+ * </ul>
+ *
+ * <h3>ES6 Module Import</h3>
+ *
+ * <code>import "@ui5/webcomponents/dist/NotificationListGroupItem.js";</code>
+ * <br>
+ * <code>import "@ui5/webcomponents/dist/NotificationAction.js";</code> (optional)
+ * @constructor
+ * @author SAP SE
+ * @alias sap.ui.webc.fiori.NotificationListGroupItem
+ * @extends sap.ui.webc.fiori.NotificationListItemBase
+ * @tagname ui5-li-notification-group
+ * @since 1.0.0-rc.8
+ * @appenddocs sap.ui.webc.fiori.NotificationAction
+ * @implements sap.ui.webc.main.IListItem
+ * @public
+ */
+@customElement("ui5-li-notification-group")
+@languageAware
+
+/**
+ * Fired when the <code>ui5-li-notification-group</code> is expanded/collapsed by user interaction.
+ *
+ * @event
```

Comment:
As part of the jsdoc changes, we need to add `sap.ui.webc.fiori.NotificationListGroupItem#toggle` so that event continues showing in the API reference